frameworks/react-introduction.md

```diff
@@ -0,0 +1,27 @@
+# React JS
+
+## Introduction
+React was created by facebook in 2013 to help with creating user interfaces. It quickly became a popular framework and many companies use it for their own web development needs. It is often used to create single page applications.
+
+## Overview
+
+React is used most in conjuction with the react router to create single page web-application that feel like multi-page websites. This can make websites feel very fast when a user does not have to wait for different pages to load. It can also reduce strain on the server by having the client render the web page they request for you. 
+
+This framework's most defining feature is it's components. React was developed to be modular, and have easy to use 'parts' that can be plugged into different parts of your application. React heavly uses a syntax called JSX, which is an html like syntax for javascript. (This can be used without react, as it is just sytatic sugar over creating DOM elements manually using `document.createElement('div')`, for example). Anything written in react is simply javascript, and can be treated as such. React stays perfomrant by utilizing what they call a virtural DOM. The framework utilizes this virtural DOM by comparing it with the real DOM to tell which elements need to be updated and which ones do not. This keeps react fast. 
```

Comment:
heavily*...  syntactic* sugar... React stays performant*....

Reply:
Thank you! Spelling isn't my best attribute!